### PR TITLE
Add DNS TXT record for ReadySetCyber-cd

### PR DESCRIPTION
**Pull request by cduhn17**
Add DNS TXT record for ReadySetCyber-cd

#  #

## 🗣 Description ##

We need to update the TXT record for the DNS to create a certificate for Let'sEncrypt. terraform-docs has been run.

Add/Update the ReadySetCyber.cyber.dhs.gov TXT

## 💭 Motivation and context ##

There are no existing ACME certs

## ✅ Pre-approval checklist ##

- [x] This PR has an informative and human-readable title.
- [x] Changes are limited to a single goal - *eschew scope creep!*
- [ ] *All* future TODOs are captured in issues, which are referenced
      in code comments.
- [x] All relevant type-of-change labels have been added.
- [x] I have read the [CONTRIBUTING](../blob/develop/CONTRIBUTING.md) document.
- [x] These code changes follow [cisagov code standards](https://github.com/cisagov/development-guide).
- [x] All relevant repo and/or project documentation has been updated
      to reflect the changes in this PR.
- [ ] Tests have been added and/or modified to cover the changes in this PR.
- [x] All new and existing tests pass.

### CODE REVIEW

File: route53_readysetcyber_app.tf [hunk 1]
```diff
@@ -46,6 +46,18 @@ resource "aws_route53_record" "ready_set_cyber_prod_AAAA" {
   zone_id = aws_route53_zone.cyber_dhs_gov.zone_id
 }
 
+resource "aws_route53_record" "readysetcyber_cd_acme_TXT" {
```
Comment (dav3r) [inline route53_readysetcyber_app.tf]: Be consistent with other RSC resource names:
```suggestion
resource "aws_route53_record" "ready_set_cyber_cd_acme_TXT" {
```

File: README.md [hunk 2]
```diff
@@ -117,6 +117,7 @@ zone.  This role has a trust relationship with the users account.
 | [aws_route53_record.ready_set_cyber_prod_acme_TXT](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/route53_record) | resource |
 | [aws_route53_record.ready_set_cyber_prod_digicert_CAA](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/route53_record) | resource |
 | [aws_route53_record.ready_set_cyber_staging_digicert_CAA](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/route53_record) | resource |
+| [aws_route53_record.readysetcyber_cd_acme_TXT](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/route53_record) | resource |
```
Comment (dav3r) [inline README.md]: Be consistent with other RSC resource names: 
```suggestion
| [aws_route53_record.ready_set_cyber_cd_acme_TXT](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/route53_record) | resource |
```

**Review by dav3r — CHANGES_REQUESTED** [review]
Minor changes requested...

**Comment by dav3r** [comment]
@cduhn17 I applied this Terraform, so you should now see that new TXT record in DNS.  Please let us know if everything looks good on your side, then we can proceed with reviewing and merging this PR.

**Comment by cduhn17** [comment]
@dav3r , 
Ive got some issues with the CAA record that is not allowing the issuance for readysetcyber-cd.crossfeed.cyber.dhs.gov

**Comment by cduhn17** [comment]
@dav3r , 
I removed this last update and going a self signed direction. This PR can be closed

**Comment by dav3r** [comment]
Per https://github.com/cisagov/cool-dns-cyber.dhs.gov/pull/99#issuecomment-2104428166, I'm closing this PR at the request of @cduhn17.

**Comment by dav3r** [comment]
Also, I went ahead and re-ran `terraform apply` from the `develop` branch of this repo in order to remove [the DNS TXT record that had been created](https://github.com/cisagov/cool-dns-cyber.dhs.gov/pull/99#issuecomment-2103275176) during the course of this PR.

**Pull request closed without merging**